scripts/build-docker.sh

```diff
@@ -79,12 +90,17 @@ if [ "${v}" != "${package_version}" ]; then
   exit 1
 fi
 
-if [ -z "${releases}" ] then
+if [ -z "${releases}" ]; then
   # stopping here after forced build
   log "no releases were configured for branch ${branch}"
   exit 0
 fi
 
+if [ "${no_tags:-}" = "true" ]; then
+  log "skip tagging as requested"
+  exit 0
```

Comment:
I would revert the if-condition and put the tagging code here in case there is further processing down the road

Reply:
changed